### DISCUSSION TITLE
Improve performance

### DISCUSSION THREAD

**Discussion by jaulz** [spatie/laravel-google-fonts#22] [Ideas]
I saw a trick at 
https://github.com/feat-agency/vite-plugin-webfont-dl#webfont-dl-vite-plugin-
that can help to improve the performance of embedded fonts by making it non-blocking. It works like this:

```html
<link rel="preload" as="style" href="/assets/webfonts.b904bd45.css">
<link rel="stylesheet" media="print" onload="this.onload=null;this.removeAttribute('media');" href="/assets/webfonts.b904bd45.css">
```

They also explain what it does in detail:
![image](https://user-images.githubusercontent.com/5358638/160902584-6c4b4d22-0e30-4a72-9624-da2ec79eab7a.png)

Do you think it could be worth to add it to this library as well?

**Comment by freekmurze** [comment]
@sebastiandedeyne what do you think?

**Reply by sebastiandedeyne** [reply]
@jaulz I link this idea. I'd be open to a PR that adds an opt-in config value to render fonts this way.

**Reply by jaulz** [reply]
@sebastiandedeyne thanks for your feedback! I just raised a corresponding PR at https://github.com/spatie/laravel-google-fonts/pull/23
